src/main/java/Duke.java

```diff
@@ -11,21 +14,29 @@ public static void main(String[] args) {
         System.out.println("-------------------------------");
         System.out.println("Hello! I'm Duke\n" +
                 "What can I do for you?");
-
-
         System.out.println("-------------------------------");
 
         String echo = " ";
+        ArrayList<String> addedList = new ArrayList<>();
```

Comment:
Perhaps can use `tasks` (plural form) as the variable name instead of addedList.